src/gui/dialogs/outro.cpp

```diff
@@ -42,15 +43,22 @@ REGISTER_DIALOG(outro)
 
 outro::outro(const game_classification& info)
 	: modal_dialog(window_id())
+	, alpha_queue_()
 	, text_()
 	, current_text_()
 	, text_index_(0)
-	, duration_(info.end_text_duration)
-	, fade_alpha_(0)
-	, fade_start_(0)
-	, fading_in_(true)
-	, timer_id_(0)
 {
+	// 3.5 seconds by default
+	const auto show_duration = info.end_text_duration > 0 ? std::chrono::milliseconds{info.end_text_duration} : 3500ms;
+
+	alpha_queue_ = {
+		{0,   255, FADE_DURATION, utils::ease_out_cubic},
+		{255, 255, show_duration, utils::ease_out_cubic},
```

Comment:
This is _clearly_ not `ease_out_cubic` if the value's remaining constant. I'd set it to `constant` or maybe `linear`.

Reply:
Already handled.